Src/MoneyFox.Ui/Views/Settings/SettingsViewModel.cs

```diff
@@ -31,4 +40,33 @@ public CurrencyViewModel SelectedCurrency
     }
 
     public IReadOnlyList<CurrencyViewModel> AvailableCurrencies { get; }
+
+    public AccountLiteViewModel? SelectedAccount
+    {
+        get => selectedAccount ??= AvailableAccounts?.FirstOrDefault(x => x.Name == (string.IsNullOrEmpty(settingsFacade.DefaultAccount) ? string.Empty : settingsFacade.DefaultAccount));
+
+        set
+        {
+            SetProperty(field: ref selectedAccount, newValue: value);
+            settingsFacade.DefaultAccount = selectedAccount != null ? selectedAccount.Name : string.Empty;
+            OnPropertyChanged();
+        }
+    }
+
+    public void LoadAccounts()
+    {
+        availableAccounts = mediator.Send(new GetAccountsQuery()).Result.Select(a => new AccountLiteViewModel(a.Name)).ToList();
+    }
+
+    public IEnumerable<AccountLiteViewModel>? AvailableAccounts
```

Comment:
I think this should be something like an IReadOnlyList or Collection. IEnumerable would have to be evaluated every time it is accessed. And I think it makes sense to have public list properties to be immutable, so there are not accidentally changed.

Reply:
Made the change as suggested.